src/store/actions.js

```diff
@@ -67,3 +69,22 @@ export const preSelectEntity = (id, selected) => ({
 export const cleanEntities = () => ({
   type: CLEAN_ENTITIES,
 });
+
+export const loadImages = (dispatch) => {
+  dispatch({
+    type: LOAD_ACTIVE_IMAGES,
+    payload: fetchActiveImages,
+    meta: {
+      notifications: {
+        rejected: {
+          variant: 'danger',
+          title: 'Can not show images data',
+          description: 'Failed receiving images from image-builder',
+        },
+      },
+    },
+    // the '.catch' part is necessary because redux-promise-middleware throws the error on REJECTED
+    // and to avoid the app exploding I need to catch it here.
+    // THANK you redux-promise-middleware for not allowing to customize this behavior. 😠
+  }).catch(() => null);
```

Comment:
Doesn't this prevent the promise-middleware from firing `REJECTED` state? What exactly is the problem? An error in console or completely broken UI?

Reply:
completely breaking UI.

If you look here: https://github.com/pburtchaell/redux-promise-middleware/blob/main/src/index.js#L162-L165
On Promise reject it dispatched REJECTED action and a line after `throw` the error again.
Looking in their examples, one way to deal with it is catching it in the action: 
https://github.com/pburtchaell/redux-promise-middleware/blob/main/examples/catching-errors/index.js#L19-L27